sdk/test/metrics/metric_instrument_test.cc

```diff
@@ -280,9 +280,9 @@ TEST(Counter, StressAdd)
   auto labelkv  = trace::KeyValueIterableView<decltype(labels)>{labels};
   auto labelkv1 = trace::KeyValueIterableView<decltype(labels1)>{labels1};
 
-  std::thread first(CounterCallback, alpha, 100000, labelkv);
-  std::thread second(CounterCallback, alpha, 100000, labelkv);
-  std::thread third(CounterCallback, alpha, 300000, labelkv1);
+  std::thread first(CounterCallback, alpha, 1000, labelkv);
+  std::thread second(CounterCallback, alpha, 1000, labelkv);
+  std::thread third(CounterCallback, alpha, 3000, labelkv1);
```

Comment:
I reduced the numbers here. This speeds up tests and makes it possible to run tests under valgrind (without running in timeouts).